tests/unit/inference/test_inference.py

```diff
@@ -209,6 +239,8 @@ def assert_fn(model_w_task):
         "token-classification": token_classification_assert,
         "text-generation": text_generation_assert,
         "text2text-generation": text2text_generation_assert,
+        "translation": translation_assert,
+        "summarization": summarization_assert
```

Comment:
The summarization task is never used - should we add a model for this task?

Reply:
I could add a second model to my unit test for summarization, but it wouldn't necessarily be testing tensor parallelism any more than before. Do you still think I should add it?

Reply:
no, that's probably not necessary. Thanks!